lib/service/storage/elasticsearch.ts

```diff
@@ -1062,11 +1191,17 @@ class ElasticSearch extends Service {
    * @returns {Promise.<{ _id, _version, _source }>}
    */
   async deleteFields(
-    index,
-    collection,
-    id,
-    fields,
-    { refresh, userId = null } = {},
+    index: string,
+    collection: string,
+    id: string,
+    fields: string,
+    {
+      refresh,
+      userId = null,
+    }: {
+      refresh?: boolean | "wait_for";
+      userId?: string;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      userId = null,
    }: Pick<KRequestParams, "refresh" | "userId"> = {},
```

With new `KRequestParams` type we can compose type like it, to DRY